src/EnergyPlus/TARCOGArgs.cc

```diff
@@ -582,13 +582,17 @@ namespace TARCOGArgs {
 					if ( thick( i ) < SlatThick( i ) ) thick( i ) = SlatThick( i );
 				} else if ( ( ThermalMod == THERM_MOD_ISO15099 ) || ( ThermalMod == THERM_MOD_CSM ) ) {
 					thick( i ) = SlatThick( i );
+					double slatAngRad = SlatAngle( i ) * 2.0 * DataGlobals::Pi / 360.0;
```

Comment:
No doubles, please use Real64.

Reply:
@vidanovic This line still needs a Real64 instead of double.

Reply:
@vidanovic This line still needs a Real64 instead of double.